consensus/parlia/parlia.go

```diff
@@ -782,6 +1150,57 @@ func (p *Parlia) FinalizeAndAssemble(chain consensus.ChainHeaderReader, header *
 	return blk, receipts, nil
 }
 
+func (p *Parlia) WithinValidatorSet(chain consensus.ChainHeaderReader, header *types.Header) bool {
```

Comment:
`WithinValidatorSet` seems not a good name, what about `IsActiveValidatorAt`, since

Reply:
Done